ISICbalance/ViewControllers/AccountViewController.swift

```diff
@@ -1,41 +1,68 @@
 //
-//  AccountView.swift
+//  AccountViewController.swift
 //  ISICbalance
 //
 //  Created by Rostislav Babáček on 11/03/2019.
 //  Copyright © 2019 Rostislav Babáček. All rights reserved.
 //
 
 import UIKit
+import SnapKit
+import SwiftKeychainWrapper
 import ACKategories
 
-class AccountView: UIView {
+class AccountViewController: BaseViewController {
+    private var viewModel: AccountViewModel
+    
+    weak var formStackView: UIStackView!
+    weak var usernameLabel: UILabel!
+    weak var usernameTextField: UITextField!
+    weak var passwordLabel: UILabel!
+    weak var passwordTextField: UITextField!
+    weak var loginButton: UIButton!
+    
+    override init() {
+        self.viewModel = AccountViewModel()
+        super.init()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        super.loadView()
+        self.view.backgroundColor = UIColor.Theme.backgroundColor
 
-    override init(frame: CGRect) {
-        super.init(frame: frame)
-        self.backgroundColor = UIColor.Theme.backgroundColor
-        
         let formStackView = UIStackView()
         formStackView.spacing = 10
         formStackView.alignment = .leading
         formStackView.axis = .vertical
-        self.addSubview(formStackView)
+        self.view.addSubview(formStackView)
+        self.formStackView = formStackView
         
         formStackView.snp.makeConstraints { (make) in
             make.centerX.equalToSuperview()
             make.leading.greaterThanOrEqualTo(20)
             make.trailing.lessThanOrEqualTo(-20)
-            make.top.equalTo(self.safeAreaLayoutGuide).offset(20)
+            make.top.equalTo(self.view.safeAreaLayoutGuide).offset(20)
         }
         
+        setupFormFields()
+        setupLoginButton()
+    }
+    
+    fileprivate func setupFormFields() {
         let usernameLabel = UILabel()
         usernameLabel.text = L10n.Login.username
+
         usernameLabel.textColor = UIColor.Theme.labelBlue
+        self.usernameLabel = usernameLabel
         formStackView.addArrangedSubview(usernameLabel)
         
         let usernameTextField = FormTextField()
+        self.usernameTextField = usernameTextField
         formStackView.addArrangedSubview(usernameTextField)
-        
         usernameTextField.snp.makeConstraints { (make) in
             make.width.equalTo(280)
```

Comment:
❕ v tomhle případě je velice špatné nastavovat šířku na pevnou velikost, lepší nastavit, `leading` a `trailing` a dát nějaký inset. Vyřešíš tím potom všechny možné velikosti obrazovek (jednotlivé iPhony), rotaci na landscape i třeba iPad

Reply:
Jak se tohle správně řeší? Nikdy jsem se u takovýchto constrainů nemohl vymotat z chyb. Prvkům toho stackview mám nastavit leading a trailing na stackview a ke každému tomuto constrainu nastavit inset?

Reply:
vždycky je možnost to nalayoutovat bez stackview 😊 v tvém případě bych smazal `formStackView.alignment = .leading` a mělo by ti to začít fungovat. Ono se defaultně použije `alignment = .fill`, což ti ve spojení s `axis = .vertical` zaručí, že se všechna vnitřní roztáhnout na celou šířku stackView

Reply:
osobní názor je takový, že to stackView cpu viewčka, která se sama dokážou nalayoutovat (roztáhnout) a já řeším pouze jejich odsazení. Většinou vůbec nepotřebuju a nechci řešit jejich výšku / šířku ve stackView

Reply:
Ok. Nastavil jsem tedy inset pro to leading/trailing stackview a textfieldům a buttonu jsem nastavil výšku. Snad jsi to takhle myslel

Reply:
ideálně bys neměl potřebovat nastavovat ani výšku textFieldu, ale může takhle zůstat 😊